pkg/commands/compute/setup/kv_store.go

```diff
@@ -107,7 +108,7 @@ func (o *KVStores) Configure() error {
 func (o *KVStores) Create() error {
 	if o.Spinner == nil {
 		return errors.RemediationError{
-			Inner:       fmt.Errorf("internal logic error: no text.Progress configured for setup.KVStores"),
+			Inner:       fmt.Errorf("internal logic error: no spinner configured for setup.KVStores"),
```

Comment:
This should have been fixed in an older PR but I think a bad rebase meant this old implementation detail was left in.